tests/runner.py

```diff
@@ -35,6 +35,9 @@ def path_from_root(*pathelems):
 except:
   raise Exception('Cannot find "COMPILER_OPTS" definition. Is %s set up properly? You may need to copy the template settings file into it.' % EM_CONFIG)
 
+# Force IDL checks
+os.environ['IDL_CHECKS'] = 'ALL'
```

Comment:
instead of putting this here, please put it in the webidl test in `test_core.py`.

but, we should also check the other 2 options (fast, and as before). but using an env var for that is a little tricky as the core test suite is built to run in parallel. how about adding this as a commandline flag, instead of an env var?